cmake/GzExport.hh.in

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2017 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+/*
+ * ==========================================================================
+ * This file was automatically generated by CMake; do not modify it directly.
+ * To modify this file, make changes to ign-cmake/cmake/Export.hh.in
+ * ==========================================================================
```

Comment:
I think this file could be a bit simpler if it just includes `Export.hh` and defines new macros with different names. We did something like this with libsdformat:

* https://github.com/ignitionrobotics/sdformat/blob/sdf12/include/sdf/system_util.hh

Reply:
Oh, I think this file shouldn't be here... It was placed here for testing the Windows CI (which still isn't resolved.)
Sorry about that! :bow: I'll revert that specific change

Reply:
The commit that introduced this file is now :boom: